TestCases/transition/E387_Airfoil/transitional_SA_LM_model_ConfigFile.cfg

```diff
@@ -0,0 +1,230 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: Eppler E387 Airfoil at Re = 200K                           %
+% Author: Andrea Rausa	                                                       %
+% Institution: Politecnico di Milano                                           %
+% Date: Dec 07th, 2022                                                         %
+% File Version 7.5.0 "Blackbird"                                               %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Physical governing equations (EULER, NAVIER_STOKES,
+%                               WAVE_EQUATION, HEAT_EQUATION, FEM_ELASTICITY,
+%                               POISSON_EQUATION)
```

Comment:
Remove all comments of this type please, keep only comments that are valuable to this specific case or to the models that it is testing.
```suggestion
```